[DOCS] Opster Migration: Add errors to troubleshooting pages (#1116)
elastic/docs-content: troubleshoot/elasticsearch/security/token-invalid-expired.md

---
applies_to:
  stack: 
  deployment:
    eck: 
    ess: 
    ece: 
    self: 

> navigation_title: "Error: token invalid or expired"

Reviewer: ```suggestion
navigation_title: "Error: Token invalid or expired"
```

Author: My bad!

---

> # Fix token invalid or expired error in Elasticsearch [token-invalid-expired]

Reviewer: ```suggestion
# Fix errors: Invalid token or token expired in {{es}} [token-invalid-expired]
```

```console
Error: token expired

Error: invalid token
```

> This error occurs when {{es}} receives a request containing an invalid or expired token during authentication. It's typically caused by missing, incorrect, or outdated tokens. If an invalid or expired token is used, {{es}} rejects the request.

Reviewer: Make it clear that these are two separate errors

Author: So strange, that doesn't match my local. I'll commit that change now

## Invalid token

> An invalid token will cause {{es}} to reject a request. Common causes include:

Reviewer: ```suggestion
{{es}} rejects requests with invalid authentication tokens. Common causes include:
```

---
applies_to:
  stack: 
  deployment:
    eck: 
    ess: 
    ece: 
    self: 
navigation_title: "Error: Token invalid or expired"
---

# Fix errors: Invalid token or token expired in {{es}} [token-invalid-expired]

```console
Error: token expired
```

```console
Error: invalid token
```

> These errors occur when {{es}} receives a request containing an invalid or expired token during authentication. It's typically caused by missing, incorrect, or outdated tokens. If an invalid or expired token is used, {{es}} rejects the request.

Reviewer: "It's" doesn't go with "These errors"...
